examples/PlotLFRE.ipynb

```diff
@@ -140,7 +140,7 @@
     "ax34.set_title(r\"$X^-$ LFRE(3-body, 4-body)\")\n",
     "ax43.set_title(r\"$X^-$ LFRE(4-body, 3-body)\")\n",
     "\n",
-    "cbar = fig.colorbar(pcm, ax=(ax34, ax43), label=\"LFRE\", location=\"bottom\")\n",
+    "cbar = fig.colorbar(pcm, ax=[ax34, ax43], label=\"LFRE\", location=\"bottom\")\n",
```

Comment:
Why is this change part of this PR?

Reply:
Third commit of the PR: [fig.colorbar wants the axes to be passed as a list, not a tuple](https://github.com/lab-cosmo/scikit-cosmo/pull/129/commits/57805762dd0e99aa5f1280841d525294da52a144)

Reply:
recent versions of matplotlib explicitly checks if this is a list when plotting